docs/quality-standards/scorecards.yaml

```diff
@@ -972,8 +972,8 @@ Remote Settings:
       documentation-operator:
         compliant: Partially
 Tecken:
-  '2023-10-06':
-    version: 1
+  '2023-12-18':
+    version: 2
     score: 50
```

Comment:
I have no idea how to do scoring--there's nothing in the docs that I could find and there's no script for it.